ngraph/core/src/op/batch_norm.cpp

```diff
@@ -67,9 +67,60 @@ void op::BatchNormInference::validate_and_infer_types()
 }
 
 std::shared_ptr<Node>
-    op::BatchNormInference::clone_with_new_inputs(const OutputVector& new_args) const
+    op::v0::BatchNormInference::clone_with_new_inputs(const OutputVector& new_args) const
 {
     check_new_args_count(this, new_args);
     return std::make_shared<BatchNormInference>(
         new_args.at(2), new_args.at(0), new_args.at(1), new_args.at(3), new_args.at(4), m_epsilon);
 }
+
+NGRAPH_RTTI_DEFINITION(op::v5::BatchNormInference, "BatchNormInference", 5);
+
+op::v5::BatchNormInference::BatchNormInference(const Output<Node>& input,
+                                               const Output<Node>& gamma,
+                                               const Output<Node>& beta,
+                                               const Output<Node>& mean,
+                                               const Output<Node>& variance,
+                                               double epsilon)
+    : Op({input, gamma, beta, mean, variance})
+    , m_epsilon(epsilon)
+{
+    constructor_validate_and_infer_types();
+}
+
+bool op::v5::BatchNormInference::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("epsilon", m_epsilon);
+    return true;
+}
+
+void op::v5::BatchNormInference::validate_and_infer_types()
+{
+    element::Type result_et;
+    PartialShape result_batch_shape;
+    PartialShape result_channel_shape; // unused here
```

Comment:
Remove?

Reply:
I cannot remove it. It used in the method infer_batch_norm_forward.